code/modules/power/supermatter/supermatter.dm

```diff
@@ -37,7 +37,7 @@
 #define HYDROGEN_TRANSMIT_MODIFIER 25 //increase the radiation emission, but less than the trit (2.5)
 #define HEALIUM_TRANSMIT_MODIFIER 2.4
 #define PROTO_NITRATE_TRANSMIT_MODIFIER 15
-#define CYRION_B_TRANSMIT_MODIFIER 20
+#define taburine_TRANSMIT_MODIFIER 20
```

Comment:
please review the changes made by your find-and-replace before making a PR, constants should have uppercased names.

Reply:
Ah thanks I'll change it